docs/conf.py

```diff
@@ -131,13 +132,24 @@
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
-# html_theme_options = {}
+html_logo = "./_static/with-text-light-190px.png"
+html_theme_options = {
+    "logo_only": True,
+    #
+    # TOC options
+    #   https://sphinx-rtd-theme.readthedocs.io/en/stable/configuring.html#theme-options
+    "includehidden": False,
+}
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-# html_static_path = ['_static']
+html_static_path = ["_static"]
 
+# A list of prefixes that are ignored for sorting the Python module
+# index (e.g., if this is set to ['foo.'], then foo.bar is shown under
+# B, not F).
+modindex_common_prefix = ["plasmapy."]
```

Comment:
Ah, I see this fixed https://plasmapy--875.org.readthedocs.build/en/875/py-modindex.html - awesome! :)

One question - it looks like a few items are just straight up missing from there. Especially, `plasmapy.simulation`. Do you have any idea what could be causing that?

Reply:
Yes, it's a larger issue I've been aware of for awhile.  Basically, not all of our modules are indexed because we basically don't `automodule` (or `:py:module:`) everything.  I've been testing a few ideas to solve this, but I'm not fully happy with any of the results yet.  It's another reason why I was diving so far into `sphinx_automodapi`.

Reply:
Oh right, it's that. Thanks!